V/VDT/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VDT"
+version = v"0.4.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/dpiparo/vdt/archive/refs/tags/v$(version).zip", "c03d21bdaaa8a0e6e557a496be505581f23aba9854e6059794721994289e97e8"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/vdt-*
+sed -i "s/-Ofast//" CMakeLists.txt
+mv Licence.txt LICENSE
+install_license LICENSE
```

Comment:
Can more simply do
```suggestion
install_license Licence.txt
```
?